src/main/scala/wordbots/Lexicon.scala

```diff
@@ -94,8 +97,11 @@ object Lexicon {
       ((S/NP)\NP,
         λ {p: TargetPlayer => λ {c: ObjectsMatchingConditions => CollectionExists(ObjectsMatchingConditions(c.objectType, c.conditions :+ ControlledBy(p)))}})
     ) +
+    ("a copy of" -> (NP/NP, λ {t:TargetObject => CopyOfC(t)})) + //can this be decomposed further?
     (Seq("cost", "energy cost") -> Seq(
       (N, Form(Cost): SemanticState),
+      (NP/Adj, λ {comp : Comparison => AttributeComparison(Cost, comp)}),//needed for "cost > x "
```

Comment:
nit- The convention I've been doing for comments is one space before and after the `//` – I think it makes them a little easier to read